tsdb/head.go

```diff
@@ -131,6 +134,8 @@ type HeadOptions struct {
 	SeriesCallback                 SeriesLifecycleCallback
 	EnableExemplarStorage          bool
 	EnableMemorySnapshotOnShutdown bool
+
+	IsolationEnabled bool
```

Comment:
You need to add this option in DB's options too, and pass it down the Head, else there is no way to disable isolation in TSDB.